### ISSUE TITLE
Prompt user to install new hooks for new languages detected on commit

### ISSUE THREAD

**Issue by JordoHeffernan** [slalombuild/secureli#493]
_As a user, I want to easily add new languages to my repository_

When a user has an existing repo with secureli installed, and they add a new language to the repo (ie add a .js file for the first time) we currently abort their commit until they run `secureli init` again. A prompt comes up in the CLI but we don't wait for input.

## AC

1. When a user commits a new language, we prompt them to install new hooks and wait for their response
2. If they input "n" we continue the commit with existing hooks
3. They they input "y" we install the new hooks, and then continue the pre-commit process with all installed hooks

**Comment by isaac-heist-slalom** [comment]
@blonienc 
I don't think that we can fix this issue as described in the AC's. The reason the commit is aborted is because the pre-commit triggers code that requests user input. From my research, git hooks don't allow user input since they are supposed to be completely automated. There are some possible workarounds, but this would involve changing the way git hooks operate, which I don't think we want to do. The pre-commit hook triggers the following securely command `secureli scan --publish-results=always`. If this command is run via the cmd line, it works as described in the AC's already, but aborts if run via pre-commit.

One possible solution is to not ask the "add new language question" during the commit. This could be done by passing a flag in with the pre-commit, such as "is-commit=true"
